tests/scripts/check-files.py

```diff
@@ -180,9 +180,9 @@ def __init__(self, log_file):
         self.check_repo_path()
         self.logger = None
         self.setup_logger(log_file)
-        self.files_to_check = (
             ".c", ".h", ".sh", ".pl", ".py", ".md", ".function", ".data",
-            "Makefile", "Makefile.inc", "CMakeLists.txt", "ChangeLog"
+            "/Makefile", "/Makefile.inc", "/CMakeLists.txt", "/ChangeLog"
+        self.extensions_to_check = (
```

Comment:
This line should go above the strings.

Reply:
Indeed, this was a bad partial commit. I rewrote it.